app/controllers/account/admin/users_controller.rb

```diff
@@ -7,7 +7,13 @@ class UsersController < ApplicationController
       before_action :admin?, only: [:change_role]
 
       def index
-        @users = User.page(params[:page]).per(20)
+        @users_scope = User.all
```

Comment:
If I`m right, firstly you get all users, and after that take only part from them. Maybe you can do this two actions in one requests. But I`m not sure. I`ve written it to draw attention of mentors, maybe somebody can give advise.